homematic/run.sh

```diff
@@ -75,5 +75,15 @@ if [ "$WIRED_ENABLE" == "true" ]; then
     WAIT_PIDS+=($!)
 fi
 
+# sync time periodically
+if [ "$RF_ENABLE" == "true" ]; then
+    while true
+    do
+        sleep 30m
+        echo "Run SetInterfaceClock"
+        "$HM_HOME/bin/SetInterfaceClock" 127.0.0.1:2001
+    done
+fi
```

Comment:
That run now in a endless loop and don't handle the docker events. Can you add a signal handler function they for SIGTERM, SIGHUB?

https://github.com/home-assistant/hassio-addons/blob/master/samba/run.sh

Reply:
of course, I will add them at the weekend.
I was thinking sleep will be interrupted from SIGTERM and SIGKILL like the wait operation at the end of the script.

Reply:
@pvizeli can you tell me what you mean with " Don't handle the docker events.".
I can stop and start the container without problems.